COMPONENTS/badge/index.css

```diff
@@ -0,0 +1,51 @@
+@import url(/CSS/global.css);
+
+html {
+  color: rgba(var(--black), 0.85);
+}
+
+.badge {
+  margin: 1rem;
+  width: max-content;
+}
+
+.badge_icon {
+  color: white;
+  padding: 4px 8.5px;
+  position: absolute;
+}
+i.badge_icon {
+  padding: 7px 11px;
+}
+
+.badge_icon.pri {
```

Comment:
Try to give the full name like instead of pri you could use primary and make it more genric.